build/teamcity/internal/release/process/make-and-publish-build-artifacts.sh

```diff
@@ -126,5 +135,13 @@ cat << EOF
 
 Build ID: ${build_name}
 
+The binaries are available at:
+  https://storage.googleapis.com/cockroach-builds-artifacts-prod/cockroach-$build_name.linux-amd64.tgz
+  https://storage.googleapis.com/cockroach-builds-artifacts-prod/cockroach-$build_name.linux-arm64.tgz
+  https://storage.googleapis.com/cockroach-builds-artifacts-prod/cockroach-$build_name.darwin-10.9-amd64.tgz
+  https://storage.googleapis.com/cockroach-builds-artifacts-prod/cockroach-$build_name.windows-6.2-amd64.zip
+
+Pull the docker image by:
+  docker pull us-docker.pkg.dev/cockroach-cloud-images/cockroachdb-customized/cockroach-customized:$build_name
```

Comment:
Let's have this print the repository where the image was pushed to.

```suggestion
  docker pull $gcr_repository:$build_name
```